pkg/bitwarden/bitwarden.go

```diff
@@ -12,11 +12,17 @@ type Attachment struct {
 	FileName string `json:"fileName"`
 }
 
+// Login represents login in BitWarden
+type Login struct {
+	Password string `json:"password"`
+}
+
 // Item represents an item in BitWarden
 // It has only fields we are interested in
 type Item struct {
 	ID          string       `json:"id"`
 	Name        string       `json:"name"`
+	Login       Login        `json:"login"`
```

Comment:
This won't exist on all BitWarden entries -- secure notes, etc don't have it. I think you may need `omitempty` here and check for `nil`

Reply:
Done.